Sources/MapboxSearch/PublicAPI/Use Cases/Place Autocomplete/PlaceAutocomplete.swift

```diff
@@ -70,13 +70,19 @@ public extension PlaceAutocomplete {
         completion: @escaping (Swift.Result<[Suggestion], Error>) -> Void
     ) {
         userActivityReporter.reportActivity(forComponent: "place-autocomplete-forward-geocoding")
-        
+
+        var navigationOptions: SearchNavigationOptions?
```

Comment:
It might be a point for discussion, but the purpose of use cases, and Place Autocomplete is one of them, was to simplify the complex configuration you need when working with a bare `SearchEngine`.

`SearchNavigationOptions` is a more complex structure, it requires more attention from the client when configuring the `options`. But it gives not that much value. It allows configuring `eta_type` with only one supported value on the server. 
It could help to speed up the request, but it won't since the `distance` calculation incurs additional latency and the ETA calculation won't make much difference in the request timing.

Reply:
The convention for our client interface in Search SDK was to make it as simple and concise as possible, so as there is no need to actually expose `SearchNavigationOptions`, I advise not to do so.

Also it's important to have complete parity with Android platform.